<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM and KNN Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="filePathEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run Analysis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="svmAccuracyLabel">
        <property name="text">
         <string>SVM Accuracy: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="knnAccuracyLabel">
        <property name="text">
         <string>KNN Accuracy: </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="svmPlotLabel"/>
      </item>
      <item>
       <widget class="QLabel" name="knnPlotLabel"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="predictionsTable">
      <property name="columnCount">
       <number>4</number>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="horizontalHeaderLabels">
       <stringlist>
        <string>Sample</string>
        <string>SVM Prediction</string>
        <string>KNN Prediction</string>
        <string>Actual</string>
       </stringlist>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>